CordovaLib/Classes/Public/CDVAppDelegate.m

```diff
@@ -78,9 +78,23 @@ - (BOOL)application:(UIApplication*)application openURL:(NSURL*)url sourceApplic
     if (!url) {
         return NO;
     }
+    
+    NSMutableDictionary * openURLData = [[NSMutableDictionary alloc] init];
+    
+    [openURLData setValue:url forKey:@"url"];
+    
+    if (sourceApplication) {
+        [openURLData setValue:sourceApplication forKey:@"sourceApplication"];
+    }
+    
+    if (annotation) {
+        [openURLData setValue:annotation forKey:@"annotation"];
+    }
 
```

Comment:
Yeah I was between this or adding the fields if they exist. I will change it to NSDictionary with literal.